test/stdlib/NSValueBridging.swift.gyb

```diff
@@ -18,11 +18,18 @@
 //
 // REQUIRES: objc_interop
 
+// The UIKit overlay isn't present on iOS 10.3.
+// UNSUPPORTED: CPU=i386
```

Comment:
What about other tests that don't depend on UIKit? Could we put these tests into a separate file that does not run on the i386 iOS simulator, while the rest of the test does?

You could also use StdlibUnittest facilities to skip a test on the simulator. See these tests: https://github.com/apple/swift/blob/30ccb529485dac910a8f2a11504205fb5ea16873/validation-test/StdlibUnittest/Common.swift#L153

Reply:
That makes sense -- I'll submit a followup PR soon.